susemanager-utils/susemanager-sls/src/tests/test_grains_virt.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from mock import MagicMock, patch
+from . import mockery
+mockery.setup_environment()
+
+from ..grains import virt
+
+@pytest.mark.parametrize("network", [True, False])
+def test_features(network):
+    """
+    test the features function
+    """
+    virt_funcs = {}
+    if network:
+        virt_funcs["network_update"] = MagicMock(return_value = True)
+    with patch.dict(virt.salt.modules.virt.__dict__, virt_funcs):
+        assert virt.features()["virt"]["enhanced_network"] ==  network
```

Comment:
Just a minor style nitpick:

```suggestion
        assert virt.features()["virt"]["enhanced_network"] == network
```

Reply:
thanks I'll fix it